pkg/util/oom/oom_linux.go

```diff
@@ -75,6 +75,7 @@ func applyOOMScoreAdj(pid int, oomScoreAdj int) error {
 			err = fmt.Errorf("failed to apply oom-score-adj to pid %d (%v)", pid, err)
 			continue
 		}
+		defer f.Close()
```

Comment:
Do we not want to return an error if the file wasn't written?  Not setting oom score would definitely qualify as an error that users should know about.